MapboxDirections/MBWaypoint.swift

```diff
@@ -17,11 +17,13 @@ open class Waypoint: NSObject, NSCopying, NSSecureCoding {
         
         It is recommended that the value of this parameter be greater than the `horizontalAccuracy` property of a `CLLocation` object obtained from a `CLLocationManager` object. There is a high likelihood that the user may be located some distance away from a navigable road, for instance if the user is currently on a driveway or inside a building.
      - parameter name: The name of the waypoint. This parameter does not affect the route but may help you to distinguish one waypoint from another.
+     - parameter allowsArrivingOnOppositeSide: A boolean value indicating whether arriving on opposite side is allowed.
      */
-    @objc public init(coordinate: CLLocationCoordinate2D, coordinateAccuracy: CLLocationAccuracy = -1, name: String? = nil) {
+    @objc public init(coordinate: CLLocationCoordinate2D, coordinateAccuracy: CLLocationAccuracy = -1, name: String? = nil, allowsArrivingOnOppositeSide: Bool = true) {
```

Comment:
Should we have a convenience initializer to make Obj-C less verbose by default?

Reply:
Seems like a good idea 👍